types/networks/src/lib.rs

```diff
@@ -134,14 +138,138 @@ struct DrandPoint<'a> {
 }
 
 /// Defines all network configuration parameters.
-#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+#[derive(Serialize, Deserialize)]
 #[serde(default)]
 pub struct ChainConfig {
     pub name: String,
     pub bootstrap_peers: Vec<String>,
     pub block_delay_secs: u64,
     pub version_schedule: Vec<UpgradeInfo>,
     pub height_infos: Vec<HeightInfo>,
+    #[serde(default = "default_policy")]
+    #[serde(with = "serde_policy")]
+    pub policy: Policy,
+}
+
+impl PartialEq for ChainConfig {
```

Comment:
That's kind of asking for trouble in the future (I'd imagine more fields will come). Probably as a really temporary workaround, it's fine, but should we open a PR adding a proper `PartialEq, Clone` to builtin-actors? I mean, if it's not exposed then it's a really bad API.